app/locales/de/messages.po

```diff
@@ -411,6 +411,7 @@ msgstr "Es ist mindestens eine numerische Massnahme erforderlich."
 msgid "controls.chart.disabled.same-unit"
 msgstr "Es sind mindestens zwei Zahlenwerte mit gleicher Einheit erforderlich."
 
+#: app/charts/index.ts
```

Comment:
I don't know why this is happening, but it also happened to me last time I used `:extract` or `:pull`

Reply:
Yeah, I am also not sure – I removed it a few times, but it keeps getting back, so I left it in the end 😅 As it's partially an auto-generated file, I think it should be fine to leave it 👍